subt/offseter.py

```diff
@@ -32,10 +33,11 @@ def run(self):
 
         while True:
             dt, channel, data = self.bus.listen()
-            if channel == "origin" and len(data) == 8:
-                robot_name, x, y, z, qa, qb, qc, qd = data
-                origin = [x, y, z]
-                offset = [o - p for o,p in zip(origin, xyz)]
+            if channel == "origin":
```

Comment:
it would be maybe worth it to add this crash source into test?

Reply:
under normal circumstances, yes... but not today